src/apps/crud/table/DefaultCRUDTable.jsx

```diff
@@ -71,18 +73,12 @@ const getColDef = (props) => {
       headerName: prop.label,
       sortable: prop.sortable,
       disableColumnMenu: prop.disableColumnMenu,
-      renderCell: ({ value }) => <Component value={value} />,
+      renderCell: prop.renderCell ? prop.renderCell : ({ value }) => <Component value={value} />,
+      width: prop.size === "flex" ? undefined : WIDTH_MAPPING[prop.size] || 50,
```

Comment:
Why this change? it is hard to read what is going on with so trinity operation and || together